eng/pipelines/coreclr/templates/build-corelib-job.yml

```diff
@@ -1,7 +1,6 @@
 parameters:
   archType: ''
   buildConfig: ''
-  compilerName: 'clang'
   condition: true
   container: ''
   crossrootfsDir: ''
```

Comment:
crossrootfsDir is not needed either, right? I expect S.P.Corelib to only be OS specific, but not arch specific, so I expect we have only two legs, one for Windows and one for Unix.

Reply:
What about this bit in Array.CoreCLR.cs:

<pre>
#if TARGET_64BIT
using nuint = System.UInt64;
#else
using nuint = System.UInt32;
#endif
</pre>

Reply:
Ah, you're right. I though it was not arch specific.